test/unit/org/apache/cassandra/index/internal/CassandraIndexTest.java

```diff
@@ -26,6 +27,7 @@
 import com.google.common.collect.*;
 import org.junit.Test;
 
+import org.apache.cassandra.Util;
```

Comment:
Nit: unused import

Reply:
Nit: this unused import is still around, it can be removed during commit

Reply:
I _did_ delete that guy :thinking: I'll push along with 3.11 later